### ISSUE TITLE
Missing Softdepend

### ISSUE THREAD

**Issue by ErythroCraft** [zhenghanlee/CustomShop#3]
Hi,

can you create softdepends to the following plugins?

• ExecutableItems to buy/sell and register created Items as self as ItemsAdder
  - [ExecutableItems](https://github.com/ssomar1607/ExecutableItems)

• Jobs Reborn to buy/sell job items to make better jobs.
  - [Jobs Reborn](https://github.com/Zrips/Jobs)

• QualityArmory to buy/sell items without the Armor GUI
  - [QualityArmory](https://github.com/ZombieStriker/QualityArmory)

• QualityArmoryVehicles2 to buy sell vehicles
- [QualityArmoryVehicles2](https://github.com/ZombieStriker/QualityArmoryVehicles2)

If not, close this issue.
THX

**Comment by zhenghanlee** [comment]
Hi there, first and foremost, thanks for the suggestions! However, I have some questions for the plugins you mentioned:

1. This is more than adding it as a soft-dependency. Some items presented in the plugin and the design of CustomShop are not compatible with each other. The plugin has items with special effects when used (when sneaking and punching) and hence not supported by customshop. (You can't set their prices in vending machines, which requires sneaking and holding the item in hand)
2. Is there a reason why we need this as a soft-dependency? Jobs isn't an economy service provider either.
3. Similar to point 1, the weapons presented in the plugin and the design of CustomShop are not compatible with each other, or at least in the first glance:
    - The plugin has special feature that changes a weapon to a scope when sneaking and hence not supported by customshop. (You can't set their prices in vending machines, which requires sneaking and holding the item in hand)
    - Although you can technically sell just one single weapon using the newt's briefcase, that isn't the shop's primary usage.
    - If sold by admin shops, the weapons will be holding onto the exact tags (clones of the original shop item), which might not be favourable for the plugin. I'm assuming there is a reason why they made it so.
4. It is a premium plugin. I'm less motivated to pay for providing support for the plugin.

In summary, point 1 and 3 requires more of a change in CustomShop's mechanics, which I am willing to rewrite my plugin to achieve. However, I have temporarily stopped the development of CustomShop (e.g. adding new features, adding support for other plugins) as I simply do not have the time now that school resumed. I will still be, however, active in fixing bugs. So keep your issues coming!

**Comment by ErythroCraft** [comment]
Ok. I understand.

**Comment by ErythroCraft** [comment]
Reopend for ideas, because the next update takes long time.
